src/server/services/wms/qgswmsutils.cpp

```diff
@@ -157,7 +157,7 @@ namespace QgsWms
         break;
       default:
         QgsMessageLog::logMessage( QString( "Unsupported format string %1" ).arg( formatStr ) );
```

Comment:
```suggestion
        QgsMessageLog::logMessage( QStringLiteral( "Unsupported format string %1" ).arg( formatStr ) );
```

Unrelated thing spotted here